src/konserve_jdbc/core.clj

```diff
@@ -197,7 +197,17 @@
     (if (:sync? env) nil (go-try- nil)))
   (-create-store [_ env]
     (async+sync (:sync? env) *default-sync-translation*
-                (go-try- (jdbc/execute! connection (create-statement (:dbtype db-spec) table)))))
+                (go-try- 
+                  ;; Using CREATE IF NOT EXISTS is regarded as a schema change. To allow the store to be used
+                  ;; where schema changes are not allowed on production e.g. planetscale or the user does have schema permissions,
+                  ;; we test for existence first. This triggers an exception if it doesn't exist which we catch. 
+                  ;; Testing for existence in other ways is not worth the effort as it is specific to the db setup 
+                  ;; not just the type
+                  (let [res (try 
+                              (jdbc/execute! connection [(str "select 1 from " table " limit 1")])
+                              (catch Exception _e (spit "trace.txt" (pr-str (.getStackTrace _e)) :append true) []))]
```

Comment:
why not just log something and return `nil` and then check on `nil?`?

Reply:
Change made. Thanks for the feedback @TimoKramer